events/event-manager.js

```diff
@@ -20,25 +20,19 @@ var createEvent = function(reqData, locals, createCallback) {
 }
 
 // This function returns an object containing the event represented by the event UID.
-var getEventByID = function(req, locals, queryCallback) {
-  // TODO (zachlefevre): Write method and remove console.log.
-  console.log(req);
-   // TODO (zachlefevre): Replace the return object with an object containing the event represented by the event UID.
-  return {
-    "event": {
-      "category" : "ENUM/STRING",
-      "contact" : "John Doe",
-      "creator" : 987654321,
-      "datetime" : "December 25 2012",
-      "description" : "Description of the holidays",
-      "location" : "Holden Hall",
-      "title" : "The Holidays"
+var getEventByID = function(eventUid, reqData, locals, queryCallback) {
+  Event.findById(eventUid, (err, event) => {
+    if(err) {
+      queryCallback(err);
+      return;
     }
-  };
-}
+      queryCallback(err, event);
+  })
+
+};
 
 // This function deletes the event represented by the event UID.
-var deleteEvent = function(req, locals, deleteCallback) {
+var deleteEvent = function(eventUid, reqData, locals, deleteCallback) {
   // TODO (zachlefevre): Write method and remove console.log().
   console.log(req);
   return {
```

Comment:
Instead of doing this, use HTTP status codes to indicate success or failure of the deletion.

``` JavaScript
    if (err) {
      res.status(400).send(err).end();
      return;
    }
    res.status(204).end();
```